code/game/objects/items/devices/radio/radio.dm

```diff
@@ -398,7 +400,7 @@ var/global/list/default_medbay_channels = list(
 	// If we were to send to a channel we don't have, drop it.
 	return null
 
-/obj/item/device/radio/talk_into(mob/living/M, message, channel, var/say_verb = "says", var/datum/language/speaking = null, var/ignore_restrained)
+/obj/item/device/radio/talk_into(mob/living/M, message, channel, var/say_verb = null, var/datum/language/speaking = null, var/ignore_restrained)
```

Comment:
default should not be changed here

Reply:
If you wanted me to reverse the change highlighted here, I've just done that.